qlty-check/src/tool/php/composer.rs

```diff
@@ -64,13 +64,15 @@ impl Tool for Composer {
     }
 
     fn extra_env_paths(&self) -> Vec<String> {
-        split_paths(
-            &std::env::var("PATH")
-                .with_context(|| "PATH not found for composer")
-                .unwrap(),
-        )
-        .map(path_to_native_string)
-        .collect_vec()
+        match std::env::var("PATH") {
+            Ok(path) => split_paths(&path)
+                .map(path_to_native_string)
+                .collect_vec(),
+            Err(_) => {
+                debug!("PATH not found for composer, using empty path");
+                Vec::new()
+            }
+        }
```

Comment:
After updating the return type for `extra_env_paths` we can update this code to return an `Err` instead of logging and returning an empty Vec